characters/playerbody.gd

```diff
@@ -1,9 +1,13 @@
 extends CharacterBody2D
 
-@export var bullet: PackedScene
+@export var shell: PackedScene
 
 const SPEED = 300.0
 const JUMP_VELOCITY = -400.0
+var aim_direction = Vector2.RIGHT
+
+func _process(delta):
```

Comment:
should be in _physics_process

Reply:
Removed lines 7 and 9 because they were unnecessary